get_started_with_gx_cloud/workshop.md

```diff
@@ -38,12 +38,28 @@ You'll need to setup a data source from GX Cloud. We provide demo data in a Post
 > **Select the `nyc_taxi_data` Data Asset**
 > 1. On the **Select tables to import** page, check the box next to `nyc_taxi_data`.
 > 1. Click **Add 1 Asset**.
+> 1. Click **Start monitoring**.
 
 <img src="img/add_demo_data_asset.png" alt="Add a Demo Data Asset" style="width:600px;"/><br>
+<img src="img/add_demo_data_monitoring.png" alt="Start Monitoring" style="width:600px;"/><br>
 
-Congratulations! You have successfully added the `nyc_taxi_data` demo Data Asset to your GX Cloud organization.
+Congratulations! You have successfully added the `nyc_taxi_data` demo Data Asset to your GX Cloud organization. Click onto the newly-created Data Asset to view it.
 
-## Create Expectations
+<img src="img/view_demo_asset.png" alt="View Demo Data Asset" style="width:600px;"/><br>
+
+## Generate Expectations
+GX Cloud helps you get started on your data quality journey by leveraging AI to analyze your data and generate Expectations for you. This is especially helpful if you're just getting started and are unsure of where to start with your data quality checks. 
```

Comment:
re "This is especially helpful if you're just getting started and are unsure of where to start with your data quality checks." I'm wary of highlighting lack of user experience as the main benefit for this feature. Could this be reworkd to highlight time-to-value for all users regardless of level of experience instead?